nemo_skills/training/data_preparation_utils/filters.py

```diff
@@ -250,20 +288,21 @@ def process_dataset_entry(self, data_entry: str) -> str:
 
 
 class CodeTextFilter(BaseParallelProcessor):
-    def __init__(self, filter_type, solution_key='generation', **kwargs):
+    def __init__(self, filter_type, prompt_config, solution_key='generation', **kwargs):
```

Comment:
Let's call it template, since that's how we reference this "config" in the rest of the codebase and docs. The prompt_config is typically referred to the "config" part that contains user/system messages

Reply:
Oh yeah, thanks for the suggestion.